overlaysFromChannelsExporter.nix

```diff
@@ -31,14 +33,23 @@ let
 
       **/
     let
-      inherit (builtins) mapAttrs attrNames concatMap listToAttrs;
+      inherit (builtins) mapAttrs foldl' filter attrNames attrValues concatMap listToAttrs elem;
       nameValuePair = name: value: { inherit name value; };
 
+      # overlays aren't system specific so arch doesn't matter here
+      channels = pkgs.x86_64-linux;
```

Comment:
Could we rather use the first element of whatever aarch is defined there? That would appear to me as a safer way which could not enter the failure mode where x86 is not among the user-supported architecture.

Reply:
I used head and attrValues to fix this and just get the first system in alphabetical order. It doesn't really matter what system it uses, just needs to get the instantiated.